src/interchange/src/json.rs

```diff
@@ -324,6 +330,7 @@ fn build_row_schema_field(
             })
         }
         ScalarType::MzTimestamp => json!("string"),
+        ScalarType::Range { .. } => json!("string"),
```

Comment:
Do we have any roundtrip tests on these JSON conversions? I don't really know enough about this code to review properly.